x-pack/plugins/infra/public/alerting/common/components/metrics_alert_dropdown.tsx

```diff
@@ -7,11 +7,10 @@
 
 import { i18n } from '@kbn/i18n';
 import React, { useState, useCallback, useMemo } from 'react';
-import {
-  EuiPopover,
-  EuiHeaderLink,
-  EuiContextMenu,
+import { EuiPopover, EuiHeaderLink, EuiContextMenu } from '@elastic/eui';
```

Comment:
Nit: We can merge eui imports: 
```suggestion
import {
  type EuiContextMenuPanelDescriptor,
  type EuiContextMenuPanelItemDescriptor,
  EuiPopover,
  EuiHeaderLink,
  EuiContextMenu,
} from '@elastic/eui';
```

Reply:
nice catch, thank you!